### PR TITLE
Set up the models

### PR DESCRIPTION
**Pull request by Patrick7321**

### CODE REVIEW

File: keyform/models.py [hunk 1]
```diff
@@ -2,5 +2,60 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.conf import settings
+from django.utils.timezone import now
```
Comment (enki211) [inline keyform/models.py]: Maybe include this
```
from django.utils.translation import ugettext_lazy as _
```
So we can translate the stuff in the tuples

File: keyform/models.py [hunk 2]
```diff
@@ -2,5 +2,60 @@
...
+
+    class Meta:
+        verbose_name_plural = 'Key Data'
+
+
+class Comment(models.Model):
+    request = models.ForeignKey(Request)
+    author = models.ForeignKey(settings.AUTH_USER_MODEL)
+    created_timestamp = models.DateTimeField(default=now, blank=True)
+    message = models.TextField()
```
Comment (enki211) [inline keyform/models.py]: You may want to add a meta class and add ordering
```python
class Meta:
    ordering = ['timestamp']
```

File: keyform/models.py [hunk 3]
```diff
@@ -2,5 +2,60 @@
...
+        ('sk', 'Stolen Key'),
+        ('wk', 'Worker Key'),
+    )
+
+    PAYMENT_TYPES = (
+        ('ca', 'Cash'),
+        ('ch', 'Check'),
+    )
+
+    building = models.ForeignKey('Building')
```
Comment (enki211) [inline keyform/models.py]: You may want to remove the quotes around Building to make it match all the other ForeignKey's

File: keyform/models.py [hunk 4]
```diff
@@ -2,5 +2,60 @@
...
+    student_name = models.CharField(max_length=128)
+    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
+    amt_recieved = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True)
+    paid_by = models.CharField(max_length=2, choices=PAYMENT_TYPES)
```
Comment (enki211) [inline keyform/models.py]: payment_method instead?

File: keyform/models.py [hunk 5]
```diff
@@ -2,5 +2,60 @@
...
+    charge_amount = models.DecimalField(max_digits=7, decimal_places=2)
+    staff_name = models.ForeignKey(settings.AUTH_USER_MODEL)
```
Comment (tom08) [inline keyform/models.py]: Maybe this field should  just be "staff" since it points to the staff user object, and not just the staff name

**Pull request merged**